src/web/core/routes/room-route.ts

```diff
@@ -1,10 +1,10 @@
 import { createRoute } from '@tanstack/react-router'
 
-import { RoomScreen } from '@/web/room'
+import { RoomScreen } from '../screens/room-screen'
 
 import { rootRoute } from './app-router'
 
-export const ROOM_PATH = '/room'
+export const ROOM_PATH = '/room/$roomId'
```

Comment:
Create a default `/room` route that will create room and redirect to `/room/$roomId` or redirect to the main in case when user not authenticated.